<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TInterface</class>
 <widget class="QMainWindow" name="TInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Интерфейс</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_param">
      <property name="text">
       <string>Параметры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_state">
      <property name="text">
       <string>Состояния</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_control">
      <property name="text">
       <string>Управление</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>315</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
